docs/pages/versions/v51.0.0/sdk/notifications.mdx

```diff
@@ -19,6 +19,8 @@ import { PlatformTags } from '~/ui/components/Tag';
 
 `expo-notifications` provides an API to fetch push notification tokens and to present, schedule, receive and respond to notifications.
 
+> **Note**: `expo-notifications` functionality is not fully supported in Expo Go. We recommend you instead use a [development build](/develop/development-builds/introduction/) to avoid limitations.
```

Comment:
```suggestion
> **warning** `expo-notifications` functionality is not fully supported in Expo Go. Instead, we recommend using a [development build](/develop/development-builds/introduction/) to avoid limitations.
```

Let's also apply the same changes from unversioned reference to here :)